src/Hit/Issue.hs

```diff
@@ -164,3 +162,24 @@ parseOwnerRepo url =
 
     stripGitSuffix :: Text -> Maybe Text
     stripGitSuffix x = whenNothing (T.stripSuffix ".git" x) (Just x)
+
+data AssignSuccess = AlreadyAssigned | NewlyAssigned
+
+assignToIssue :: Issue -> IO ()
+assignToIssue issue = withOwnerRepo assignAction >>= \case
+    Left err              -> errorMessage (T.pack $ show err) >> exitFailure
+    Right AlreadyAssigned -> successMessage "Already assigned to issue"
+    Right NewlyAssigned   -> successMessage "Assigned to issue"
```

Comment:
Let's add the issue number and name in here (ideally the link as well, so the user could just click and go there).

Reply:
I think this is a great idea but would like that change to be an separate issue / pr.